.packit.yml

```diff
@@ -0,0 +1,34 @@
+upstream_package_name: subscription-manager
+downstream_package_name: subscription-manager
+specfile_path: subscription-manager.spec
+upstream_tag_template: "subscription-manager-{version}-1"
+
+srpm_build_deps:
+  - python3
+  - tito
+
+actions:
+  pre-sync:
+    - tito build --test --tgz
+
+files_to_sync:
+  - src:
+    - subscription-manager.spec
+    - "subscription-manager-*.tar.gz"
+    dest: .
```

Comment:
ditto: `files_to_sync` is mostly for upstream -> downstream, which is not what is done in this case